index.php

```diff
@@ -1,23 +1,48 @@
-<?php
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Scrabble</title>
+</head>
+<body>
+    <h1>Scrabble</h1>
 
-use Src\Boot;
-use Src\Engine\Dictionary\Dictionary;
-use Src\Engine\Scrabble;
+    <form method="post" action="<?php echo $_SERVER["PHP_SELF"]; ?>">
+        <label for="rack">Enter your Scrabble tiles:</label>
+        <input type="text" name="rack" id="rack" required>
+        <br>
+        <input type="submit" value="Submit">
+    </form>
 
-require_once 'Src/Boot.php';
+    <?php
+    use Src\Boot;
+    use Src\Engine\Dictionary\Dictionary;
+    use Src\Engine\Scrabble;
 
-$boot = new Boot();
+    require_once 'Src/Boot.php';
 
-$dictionary = new Dictionary($boot);
+    if ($_SERVER["REQUEST_METHOD"] == "POST") {

```

Comment:
it's good practice to validate your inputs
```suggestion
    if (
        $_SERVER["REQUEST_METHOD"] == "POST" &&
        array_key_exists('rack', $_POST) &&
        is_array($_POST['rack']) &&
        ! empty($_POST['rack'])
    ) {

```